strategy/rfp-to-videographers.md

```diff
@@ -1,154 +1,114 @@
+Approach
 
-Goals
-
-- Shock the audience
-- Lead to introspection and questioning
+- Shock the audience without imposing an agenda or lecturing the target audience
+- Lead to introspection and questioning by asking people if they are comfortable with the impact of these forms of technology on their lives
 - Make the audience think and reach their own conclusions
-- Provoke, rather than convince
-
-Audience
+- Provoke to reflect and reconsider
 
-All ages from 16
+#### "Reality Shock" Campaign 
 
-Description
+Series of short, dramatic videos that illustrate the pervasive use of technology and rapid, drastic changes in our society and provoke people to ask, "Is this really acceptable in a healthy, fair, and strong society?"
 
-- Argumentation, reasoning and "lectures" only go so far, hence the pressing need to first trigger an "emotional" response.
-- The campaign will feature short videos (2-3 minutes each) that aim to be dramatic and impactful through the combined use of words, cinematography and music.It does not aim to explicitly pursue a specific agenda, push on the audience a clear message, or impose a "lecture". 
-- Rather, it aims to provoke introspection by telling provocative stories, the conclusions of which are left to the audience. 
-- Why is it important to adopt a "neutral" approach based on providing factual observations in a spectacular and dramatic form? Because this campaign is based on the principle that we will not change our behaviors on the mere basis of facts and reason if we have no motivation to challenge our thinking in the first place and take a step back from our habits. The dramatic videos aim to shock and leave a strong impression that ideally will lead to wondering where each of us stands in all this.
+Approach
 
-#### Video concept and script
+Argumentation, reasoning, and "lectures" only go so far to change people's mind and behaviors; hence we propose to trigger an "emotional" response that leads to reflection. The campaign will feature short videos (2-3 minutes each) that aim to be dramatic and impactful through the combined use of words, cinematography, and music. It does not aim to pursue a specific agenda, impose on the audience a particular message, or impose a "lecture." Rather, it aims to provoke introspection by telling provocative stories, the conclusions of which are left to the audience. Why is it important to adopt a "neutral" approach based on providing factual observations in a spectacular and dramatic form? Our belief is that people will not change their behaviors on the mere basis of facts and reason if they have no motivation to challenge their thinking in the first place. The dramatic videos aim to shock and leave a strong impression that ideally will lead to reconsideration of our unconscious and pervasive use of technology.
```

Comment:
Suggest breaking this very heavy paragraph into several, otherwise will not be read. Does not have to be bullet points.